src/Controller/ModulesController.php

```diff
@@ -321,8 +321,9 @@ public function saveJson(): void
             $this->Modules->saveRelated($objectId, $this->objectType, $relatedData);
         } catch (BEditaClientException $error) {
             $this->log($error, LogLevel::ERROR);
```

Comment:
Currently the `error` object received in the Vue app is empty. I suggest this change in order to get a meaningful response and avoid the subsequent `/dashboard/messages` ajax call that can be avoided 


```php
} catch (BEditaClientException $ex) {
            // this has the same effect as the current log
            $this->log($ex->getMessage(), LogLevel::ERROR);
            // meaningful messages received by API (i.e. `error` response is available in exception attributes)
            // if `error` is empty a generic error message can be displayed
            $this->set(['error' => $ex->getAttributes()]);
            $this->set('_serialize', ['error']);

            return;

```